<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1137</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Кол-во переменных</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Функция отклика</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="function">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>1+u1+u2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Кол. экспериментов</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Помехи</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="d">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>D</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>M</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>0.5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Базовые точки</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>u1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="u1_del">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="u2_del">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="u3_del">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>310</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>u2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>u3</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="u2_int">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>390</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Интервалы покачивания</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>u1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="u3_int">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>420</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>u3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>u2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="u1_int">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>360</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Линейная модель</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Квадратичная модель</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Адекватность модели</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Квадратичная</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Линейная</string>
   </property>
  </widget>
  <widget class="QLabel" name="quad_model_adeq">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>160</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="linear_model_adeq">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Лин</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fish_lin">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>1.43</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="stud_lin">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>290</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>0.68</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Помехи</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>290</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Стьюдент</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>260</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Фишер</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="koh_lin">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>320</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>6.2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>320</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Кохрена</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="stud_quad">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>290</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>0.68</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fish_quad">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>260</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>1.43</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="koh_quad">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>320</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>6.2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_29">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>240</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Кв</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calc_btn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>360</y>
     <width>501</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>Вычислить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>240</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Лин</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>260</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Квад</string>
   </property>
  </widget>
  <widget class="QLabel" name="linear_model_adeq_d">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>240</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="quad_model_adeq_d">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>260</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_30">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>220</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дисперсия адекватности</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="exp_num">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>15</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="num2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;2</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="num3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;3</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="linear_model">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>741</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="quad_model">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>70</y>
     <width>711</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
